x/staking/types/validator.go

```diff
@@ -88,3 +95,29 @@ type ValidatorDelegations struct {
 	Height               int64
 	Timestamp            time.Time
 }
+
+//-----------------------------------------------------
+
+//ValidatorCommission allow to build a validator commission instance
+type ValidatorCommission struct{
+	ValAddress sdk.ValAddress
+	Commission 			int64
+	MinSelfDelegation int64
+	Height               int64
+	Timestamp            time.Time
+}
+
+//NewValidatorCommission return a new validator commission instance
+func NewValidatorCommission(ValAddress sdk.ValAddress,Rate int64,
+	MinSelfDelegation int64,Height int64,Timestamp time.Time)(ValidatorCommission){
+	return ValidatorCommission{
+		ValAddress    :ValAddress,
+		Commission 		:Rate,
+		MinSelfDelegation:MinSelfDelegation,
+        Height        :Height,
+        Timestamp     :Timestamp,
+	}
+}
+
```

Comment:
I think we should implement the `Equal` method for this new type. This would be probably useful inside the tests